packages/ui/src/common/components/MarkdownPreview/MarkdownPreviewStyles.tsx

```diff
@@ -11,11 +11,11 @@ export interface MarkdownPreviewStylesProps {
 const normalFontSize = ({ size }: MarkdownPreviewStylesProps) => {
   switch (size) {
     case 'xs':
-      return '12px'
-    case 's':
       return '14px'
+    case 's':
+      return '18px'
     default:
-      return '16px'
+      return '20px'
```

Comment:
Maybe a bit much? Also text in tables is different now.

![fontsize](https://user-images.githubusercontent.com/31551045/227800131-9b35fe03-266c-4d84-b36a-24563d169ff3.png)

Probably hard to find one setting that fits all. Other pages use a dropdown to change the font size depending on user needs. Or something PDF viewers have:
![size](https://user-images.githubusercontent.com/31551045/227800310-285abfaf-71fb-4157-a558-df8fd976c3e4.png)

Reply:
thanks, I will fix the table font issue in this PR.
https://github.com/Joystream/pioneer/pull/4265

Let's merge this one and create a new issue for font size selection dropdown.

Reply:
Ok. I assume it's still too big for some users. Also we are wasting space left and right, especially a problem on mobile (=> followup).

![font](https://user-images.githubusercontent.com/31551045/228529808-88df0c42-acf7-406b-844b-00c1e8e58aef.png)